app/controllers/items_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ItemsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @list = List.find(params[:list_id])
+    @user = current_user
+    @item = @list.items.build(item_params)
+    @item.list = @list
```

Comment:
Line 8 can be removed, `@item.list` is already populated correctly thanks to line 7.